src/cleanup.c

```diff
@@ -173,13 +176,25 @@ clean_up_dir(struct conf *conf, const char *dir, float limit_multiple)
 	traverse(dir, traverse_fn);
 
 	// Clean the cache.
+#ifdef _WIN32
+	cc_log("Before cleanup: %lu KiB, %I64u files",
```

Comment:
Will it fit ? I suppose we could cast it to float (without decimals) or something, for simplicity

Reply:
Doesn't really matter since it's just a debug log message, I'd say. Also, it's extremely unlikely that somebody has a cache with more than 2<sup>32</sup> files.

Reply:
Casting to float means that it will be incorrect after [2<sup>24</sup> + 1 = 16,777,217](https://stackoverflow.com/a/3793950/331771) files, which a large cache might contain.